roles/gcp_setup_credentials/README.md

```diff
@@ -0,0 +1,52 @@
+# aws_setup_credentials
+
+A role to define credentials for google.cloud modules. The role defines a variable named **gcp_setup_credentials\_\_output** which contains GCP credentials for google.cloud modules based on user input.
+
+## Requirements
+
+N/A
+
+## Role Variables
+
+- **gcp_auth_kind**: The type of credential to use. Choices: ``application``, ``machineaccount``, ``serviceaccount``, ``accesstoken``. Default to ``serviceaccount``
```

Comment:
Do we also want to pull these credentials directly from the ENV variables set by the AAP Google Compute Engine credential type?

Reply:
Done!!
Support for env var has been added